### PR TITLE
Implemented "Back to Top" button

### PR DESCRIPTION
**Pull request by adityarai0705**
It will appear after 700px has been scrolled and on clicking it, the page will be scrolled back to the top.

### CODE REVIEW

File: client/src/components/ButtonBacktoTop/ButtonBacktoTop.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.TopButton {
```
Comment (Ashish-Verma-MNNIT) [inline client/src/components/ButtonBacktoTop/ButtonBacktoTop.css:1]: You should do it with tailwind instead of making a new css file.

**Review by Ashish-Verma-MNNIT — CHANGES_REQUESTED** [review]
Just A minor change...

File: client/src/components/ButtonBacktoTop/ButtonBacktoTop.css (hunk 1 above)
Comment (adityarai0705) [reply client/src/components/ButtonBacktoTop/ButtonBacktoTop.css:1]: Okay,

**Review by Ashish-Verma-MNNIT — APPROVED** [review]
Good job 👍🏻

**Review by sanskaromar — APPROVED** [review]
👍

**Pull request merged**